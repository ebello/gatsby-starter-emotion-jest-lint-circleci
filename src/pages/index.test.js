import React from 'react';
import { render } from '@testing-library/react';
import Home from './index';

describe('Home', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
