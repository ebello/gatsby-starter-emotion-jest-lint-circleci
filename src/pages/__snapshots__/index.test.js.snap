// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  padding: 32px;
  background-color: hotpink;
  font-size: 24px;
  border-radius: 4px;
}

<div
    class="emotion-0"
  >
    Hello world!
  </div>
  .emotion-0 {
  margin-left: 18px;
  padding: 12px;
}

<div
    class="emotion-0 emotion-1"
  >
    this is a description
  </div>
</DocumentFragment>
`;
